subprojects/version-control/src/test/groovy/org/gradle/vcs/git/internal/GitVersionControlSystemSpec.groovy

```diff
@@ -33,14 +34,14 @@ class GitVersionControlSystemSpec extends Specification {
     private RevCommit c1
     private RevCommit c2
 
-    @Rule
     TestNameTestDirectoryProvider tmpDir = new TestNameTestDirectoryProvider()
-
-    @Rule
     GitRepository repo = new GitRepository(tmpDir.getTestDirectory())
+    GitRepository repo2 = new GitRepository(tmpDir.getTestDirectory().file('other'))
```

Comment:
nit, I think you could also do `new GitRepository("other", tmpDir.testDirectory)`